### DISCUSSION TITLE
Error playing on web

### DISCUSSION THREAD

**Discussion by Cypher140** [bluenviron/mediamtx#3890] [Questions]
### Question

Good morning, I use MediaMTX and I find it great. the performance is there and I congratulate you on your work. However, I have been experiencing difficulties connecting to the Internet for some time now. in fact, the page is displayed with the usual error message but the flow once launched is not displayed. There is detection but it seems that a few things block it and come back to the no flow error message. test with several browsers ditto. I saw a turn/stun problem but I use the fixed IP address of my server. any ideas???

**Comment by aler9** [comment]
This is impossible to debug if you don't provide server configuration and server logs. If you still have problems, open another issue and provide precise instructions on how to replicate the issue.
